client/rpcserver/handlers.go

```diff
@@ -452,4 +461,71 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	exchangesRoute: {
+		pwArgsShort: ``,
+		argsShort:   ``,
+		cmdSummary:  `List assets supported by all registered DEX servers their and active trades.`,
```

Comment:
How about `Detailed information about known exchanges, markets, and active trades.`